nss-evening-cohort-14/SweetTooth
# Jdc m3 ud payment

## Description
Update/Softdelete  a PaymentMethod

## Related Issue

#76 

## Motivation and Context
User wants to be able to update and delete their Payment methods

## How Can This Be Tested?
`git fetch origin jdc-m3-UD-payment`
- Navigate to the UserProfile view, and any current payment methods should appear on screen
(or add a new payment method)
- Confirm edit works, then try to delete -- softDelete should be toggled to true in your localDB

## Types of changes
